redis/src/test/java/org/apache/calcite/adapter/redis/RedisCaseBase.java

```diff
@@ -36,16 +43,33 @@
   public static final String HOST = "127.0.0.1";
   private static final String MAX_HEAP = "maxheap 51200000";
 
+  // If Docker is running, a container will be used
+  public static final GenericContainer<?> REDIS_CONTAINER =
+      new GenericContainer<>("redis").withExposedPorts(6379);
```

Comment:
consider using a fixed version image, for reproducible tests

Reply:
I agree, let's freeze the version ideally using the same that is present in the gradle.properties.